server/src/UseCases/Common/Result/ResultBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+
+namespace BGS.UseCases.Common.Result;
+
+public class ResultBuilder
+{
+    public static Task<Result> BuildSucceed()
+    {
+        return Task.FromResult(new Result
+        {
+            IsSucceeded = true,
+        });
+    }
+    
+    public static Task<Result> BuildFailed(string message)
```

Comment:
Method `BuildFailed` is never used

Reply:
removed for now, but this method will back in next taks (update and delete)